### ISSUE TITLE
Masks For Object Detection Labels

### ISSUE THREAD

**Issue by kottav** [roflcoopter/viseron#340]
Support for masks for individual object detection labels would be fantastic. This would enable MQTT alerts for example when a person is in the driveway but not when the parked cars are in the driveway because the cars would have an object label mask for the car label.

I like to use the MQTT binary sensor states for automations. I do have require_motion set to True for the car object detection label, but it doesn't always work perfectly such as when headlights from cars on the road shine on the paint of the parked driveway cars which makes viseron think there is motion thus defeating the require_motion flag and creating a false alert and recording.

Keep up the great project! I'm loving it.

**Comment by roflcoopter** [comment]
Great idea!
Sounds fairly simple to implement.

It's gonna take me a little while tho, I would like to finish off #306 first before I take on anything new.

Updating docs and tests is what remains of Viseron v2

**Comment by kottav** [comment]
I realized what I'm trying to do may be achievable with the current code by using zones with specific object detection labels.

I want to ignore the parked cars in the front of the driveway Zone while still detecting people in that zone. 

In the rear of driveway Zone I want to detect cars and 
people. How do I setup the config to detect people in all zones, but exclude cars from one particular zone?

I've tried configs like this:

Cameras
  Name: driveway camera
  Motion:
    Mask:.  #<Motion mask to ignore traffic on street>
        Points
   Zones:
      Front_of_driveway
         Points
         Labels
            - label: person
     rear_of_driveway
        Points
   Object_detection:
        Labels:
            - label: person
              <Label parameters>
            - label: cars
              <Label parameters>
Global_object_detection:
  Labels:
      - label: person
           <Global_label_parameters>
      - label: car
           <Global_label_parameters>

**Comment by kottav** [comment]
I figured it out. The trick was to have the object_detection labels in the individual camera config be restricted to just people. Then in the zone labels for that camera add in car to the zones you want to detect cars in. From my example I wanted to detect cars when they enter the driveway in the rear_of_driveway zone. So I added the label car to the rear_of_driveway zone while all other parts of the frame only look for people.
